### ISSUE TITLE
Translating emoji results in: JSONDecodeError

### ISSUE THREAD

**Issue by louwers** [ssut/py-googletrans#147]
```py
from googletrans import Translator
translator = Translator()
translator.translate('this does not work 😭')
```

Will result in:
```
*** json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```
I can see it results in a 403

Similar problem, now closed:
https://github.com/terryyin/translate-python/issues/27

**Comment by H0bby** [comment]
Try this  PR [https://github.com/ssut/py-googletrans/pull/134](https://github.com/ssut/py-googletrans/pull/134).

**Comment by louwers** [comment]
Works! But the emoji is not included in the translation. Don't know if it gets removed on Google's side?

**Comment by H0bby** [comment]
Emoji in translation  is  Schrodinger's cat . Sometimes include, but sometimes not.  🤷‍♂️

**Comment by qinst64** [comment]
the pypi version is still [2.4.0 released at Dec2018](https://pypi.org/project/googletrans/#history)
thus `translator.detect('😀')` raises __JSONDecodeError__
pls update

**Comment by rockdrigoma** [comment]
Same for me trying to use batch translation with emojis:

`translator.translate(['hola','adios', '💚⚽️'], src='es', dest='en')`

```
raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```
